packages/env/lib/init-config.js

```diff
@@ -146,6 +212,76 @@ function getBaseDockerImage( phpVersion, isCLI ) {
 	return wordpressImage + phpVersion;
 }
 
+/**
+ * Generates content for the Dockerfile to install dependencies.
+ *
+ * @param {string}   environment The kind of environment that we're installing dependencies on ('wordpress' or 'cli').
+ * @param {WPConfig} config      The configuration object.
+ *
+ * @return {string} The Dockerfile content for installing dependencies.
+ */
+function installDependencies( environment, config ) {
+	let dockerFileContent = '';
+
+	// At times we may need to evaluate the environment. This is because the
+	// WordPress image uses Ubuntu while the CLI image uses Alpine.
+
+	// Start with some environment-specific dependency installations.
+	if ( environment === 'wordpress' ) {
+		dockerFileContent += `
+# Make sure we're working with the latest packages.
+RUN apt-get -qy update
+
+# Install some basic PHP dependencies.
+RUN apt-get -qy install $PHPIZE_DEPS && touch /usr/local/etc/php/php.ini
+
+# Set up sudo so they can have root access.
+RUN apt-get -qy install sudo
+RUN echo "$HOST_USERNAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers`;
```

Comment:
Since the sudo command is shared, it could go into one of the shared sections below

Reply:
I feel like it makes sense to keep it separate since `sudo` is being installed in a platform-specific way. The `echo` would be the only duplication here and it feels like it would be strange by itself. I'm not willing to die on this hill though :smile:

Reply:
Sounds good 👍